guides/v2.3/graphql/mutations.md

```diff
@@ -0,0 +1,136 @@
+---
+group: graphql
+title: Mutations
+---
+
+While GraphQL queries perform read operations, mutations change the data. A mutation can create, update, or delete objects and fields. In REST terminology, queries operate like `GET` requests, while mutations are similar to `POST`, `PUT`, and `DELETE`.
+
+A mutation contains the following elements:
+
+* The keyword `mutation`
+* An operation name for your local implementation. This name is required if you include variables. Otherwise, it is optional.
+* The mutation name
+* The input object or attributes. Most mutations require an input object that contains data or individual attributes for the Magento server to process. However, some mutations, such as `createEmptyCart`, do not require an input object. In this particular case, the authorization token passed with the request provides the needed context.
+* The output object, which specifies which data the mutation returns. 
+
+The following example shows the structure of the `createCustomer` mutation:  
+
+```text
+mutation myCreateCustomer{
+    createCustomer(
+        input: CustomerInput!): CustomerOutput
+    }
+}
+```
+
+In this example, `myCreateCustomer` identifies your implementation.  `CustomerInput` is a non-nullable object that defines a customer. (The exclamation point indicates the value is non-nullable.) `CustomerOutput` defines which fields to return. 
+
+Now let's take a look at a fully-defined mutation. This time, we'll specify the minimum fields needed to create a customer (`firstname`, `lastname`, `email`, and `password`). We could include the same fields in the output, but GraphQL allows you to return only the data you need, which is the customer `id`.
```

Comment:
"This time, we'll specify the minimum fields needed as input to create a customer..."

Since we are defining the inputs/outputs of a mutation I thought we should be explicit here in saying these are the inputs. But maybe that's obvious.